Source/Core/Core/Movie.cpp

```diff
@@ -72,6 +72,8 @@
 
 // The chunk to allocate movie data in multiples of.
 #define DTM_BASE_LENGTH (1024)
+#define DTM_GC_FRAME_LENGTH (sizeof(ControllerState) + sizeof(u64))
+#define DTM_WII_FRAME_LENGTH (size + sizeof(u64))
```

Comment:
```suggestion
static constexpr size_t DTMWiiFrameLength(size_t size) {
  return size + sizeof(u64);
}
```

Otherwise this is implicitly dependent on `size` and can break if code is ever moved around in the future (this function should also be placed in the Movie namespace)